server/s3_bucket_metadata.cc

```diff
@@ -279,6 +279,32 @@ std::string S3BucketMetadata::get_tags_as_xml() {
   return tags_as_xml_str;
 }
 
+std::string S3BucketMetadata::get_bucket_versioning_status_as_xml() {
+  s3_log(S3_LOG_INFO, stripped_request_id, "%s Entry\n", __func__);
+  std::string versioning_status;
+  std::string versioning_status_as_xml_str;
+
+  if (bucket_versioning_status != "Unversioned") {
```

Comment:
"Unversioned"
for versioning can we have some key val defied class

it seems hardcoded

Reply:
I agree. But this is how it has been implemented and changing would need an effort for existing code as well.

Reply:
Resolved.

Not having access to resolve comment